src/core/locator/qgslocatorfilter.h

```diff
@@ -198,6 +198,11 @@ class CORE_EXPORT QgsLocatorFilter : public QObject
      */
     virtual QString displayName() const = 0;
 
+    /**
+     * Returns a translated, description for the filter.
+     */
```

Comment:
\since

Reply:
No strong opinion - if you need it in 3.18, go ahead

Reply:
3.20 it is.